### DISCUSSION TITLE
Generater测例float报错fail: not equ

### DISCUSSION THREAD

**Discussion by 91Mrcui** [arcsysu/SYsU-lang#107] [Q&A]
请问这个报错信息是什么意思？
![image](https://github.com/arcsysu/SYsU-lang/assets/87711064/4635c13a-9d7e-442e-bd43-9b2ca656b650)

**Comment by zejia-lin** [comment]
程序输出结果与标准答案不一致，如果感兴趣，可以在[这里](https://github.com/arcsysu/SYsU-lang/blob/c4ea7b15fe259add28dbc2cb6bd7186677c96caa/compiler/sysu-compiler#L349)把输出打出来看看
